index.html

```diff
@@ -189,7 +247,8 @@ <h4 class="lang-text">Skills</h4>
 
 </div>
 </section>
-<footer>
+
+<div class="contact-me">
```

Comment:
- [ ] I kindly suggest that you follow the Figma design for your project because it's a part of the project requirements.
- [ ] Kindly make sure your bold text aligns with your form and make your design match the Figma template design as much as possible

| Your Design  | Template Design |
| ------------- | ------------- |
| ![Screenshot from 2022-05-07 23-49-49](https://user-images.githubusercontent.com/52934128/167274398-627e9880-0171-446a-b576-7ae64f277d12.png)|![Screenshot from 2022-05-07 23-50-11](https://user-images.githubusercontent.com/52934128/167274384-06942501-7415-46b4-880c-07e865cf621c.png) |